### ISSUE TITLE
Provide several entry starting points - virtual real estate around these points will be more valuable?

### ISSUE THREAD

**Issue by stefek99** [decentraland/proposals#7]
When I launch app - https://decentraland.org/app/ - it always starts at the same location.

I couldn't find anything in the whitepaper - https://decentraland.org/whitepaper.pdf - about the place where I start...

Because objects located closers to the starting point are more likely to attract attention, I'm suggesting some model where there will be multiple starting points...

I'm not sure about the exact mechanics, maybe this problem is already solved?

**Comment by randomer666** [comment]
Thats a great idea actually, focusing the creative force around different entry points will make the world much more interesting. Not to mention the economical profit)

**Comment by pookage** [comment]
Agreed - my initial thought would be that it would work similarly to an MMO in which each user is associated with a particular region and they spawn there?  It could be adjusted to ensure an even distribution of 'start locations'.  We wouldn't want it to be fully random otherwise people would never know where they were and that sense of geographical community wouldn't emerge.
